src/sentry/api/endpoints/sentry_app_details.py

```diff
@@ -14,11 +14,11 @@
 
 
 class SentryAppDetailsEndpoint(SentryAppBaseEndpoint):
-    def get(self, request, sentry_app):
-        return Response(serialize(sentry_app, request.user, access=request.access))
+    def get(self, request, sentry_app, access):
+        return Response(serialize(sentry_app, request.user, access=access))
 
     @catch_raised_errors
-    def put(self, request, sentry_app):
+    def put(self, request, sentry_app, **kwargs):
```

Comment:
Adding these `**kwargs` changes isn't supposed to have any side-effects right? Is it just a design pattern that wasn't there before?

Reply:
🙃